src/librustc_mir/tcx/expr.rs

```diff
@@ -271,44 +287,47 @@ impl<'tcx> Mirror<'tcx> for &'tcx hir::Expr {
             }
 
             // Now comes the rote stuff:
-
-            hir::ExprRepeat(ref v, ref c) =>
-                ExprKind::Repeat { value: v.to_ref(), count: c.to_ref() },
-            hir::ExprRet(ref v) =>
-                ExprKind::Return { value: v.to_ref() },
-            hir::ExprBreak(label) =>
-                ExprKind::Break { label: label.map(|_| loop_label(cx, self)) },
+            hir::ExprRepeat(ref v, ref c) => ExprKind::Repeat {
+                value: v.to_ref(),
+                count: c.to_ref(),
+            },
+            hir::ExprRet(ref v) => ExprKind::Return { value: v.to_ref() },
+            hir::ExprBreak(label) => ExprKind::Break { label: label.map(|_| loop_label(cx, self)) },
             hir::ExprAgain(label) =>
                 ExprKind::Continue { label: label.map(|_| loop_label(cx, self)) },
```

Comment:
it's not clear to me what distinguishes this case from the cases above? I preferred the original, more consistent form overall.